lib/model/QubitInformationObject.php

```diff
@@ -2599,25 +2604,25 @@ public function setPhysicalObjectByName($physicalObjectName, $options)
     }
   }
 
-  public function importPhysicalObject($location, $name = false, $type = false, $label = false)
+  public function importPhysicalObject($location, $name, $options = array())
   {
-    if ($label && $type)
+    if ($options['label'] && $options['type'])
```

Comment:
Is there a case where this ever gets called when these option elements don't exist? If so, we should check isset first right?